locales/nb-no.json

```diff
@@ -0,0 +1,797 @@
+      "fail": "'aria-labelledby'-attribut eksisterer ikke, eller refererer elementer, som ikke eksisterer - eller refererer til elementer, som er tomme"
+    },
+    "avoid-inline-spacing": {
+      "pass": "Ingen inline styling med '!important', som påvirker tekst-mellomrom-avstand er spesifisert",
+      "fail": {
+        "singular": "Fjern '!important' fra inline stylings ${data.values}, da overskrivning av dette ikke er understøttet i de fleste nettlesere",
+        "plural": "Fjern '!important' fra inline styling ${data.values}, da overskrivning av dette ikke er understøttet i de fleste nettlesere"
+      }
+    },
+    "button-has-visible-text": {
+      "pass": "Elementet har indre tekst, som er synlig for skjermlesere",
+      "fail": "Elementet har ingen indre tekst, som er synlig for skjermlesere"
+    },
+    "doc-has-title": {
+      "pass": "Dokumentet har et <title> element som ikke er tomt",
+      "fail": "Dokumentet mangler et <title> element med innhold"
+    },
+    "exists": {
+      "pass": "Elementet eksisterer ikke",
+      "fail": "Elementet eksisterer"
+    },
+    "has-alt": {
+      "pass": "Elementet har en 'alt'-attributt",
+      "fail": "Elementet har ikke en 'alt'-attributt"
+    },
+    "has-visible-text": {
+      "pass": "Elementet har tekst som er synlig for skjermlesere",
+      "fail": "Elementet har ikke tekst som er synlig for skjermlesere"
+    },
+    "is-on-screen": {
+      "pass": "Elementet er ikke synlig",
+      "fail": "Elementet er synlig"
+    },
+    "non-empty-alt": {
+      "pass": "Elementet har 'alt'-attributt med innhold",
+      "fail": "Elementet har ingen 'alt'-attributt, eller 'alt'-attributten er tom"
+    },
+    "non-empty-if-present": {
+      "pass": {
+        "default": "Elementet har ikke en 'value'-attributt",
+        "has-label": "Elementet har en 'verdi'-attributt med innhold"
+      },
+      "fail": "Elementet har en 'value'-attributt, og 'verdi'-attributten er tom"
+    },
+    "non-empty-title": {
+      "pass": "Elementet har en 'title'-attributt",
+      "fail": "Elementet har ingen 'title'-attributt, eller 'title'-attributten er tom"
+    },
+    "non-empty-value": {
+      "pass": "Elementet har 'value'-attributt med innholdet",
+      "fail": "Elementet har ingen 'value'-attributt, eller 'value'-attributten er tom"
+    },
+    "role-none": {
+      "pass": "Elementets standard semantikk ble overskrevet med attributten 'role=\"none\"'",
+      "fail": "Elementets standard semantikk ble ikke overskrevet med attributten 'role=\"none\"'"
+    },
+    "role-presentation": {
+      "pass": "Elementets standard semantikk ble overskrevet med attributten 'role=\"presentation\"'",
+      "fail": "Elementets standard semantikk ble ikke overskrevet med attributten 'role=\"presentation\"'"
+    },
+    "caption-faked": {
+      "pass": "Den første række i tabellen er ikke brukt som en beskrivelse ('caption')",
```

Comment:
Original:"The first child of the table should be a caption instead of a table cell"
I suggest: 
"Den første raden i tabellen er ikke brukt som en beskrivelse ('caption')"